sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -1,136 +1,339 @@
+### FormRecognizerClient
 
-To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below,
-or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
+`FormRecognizerClient` provides operations for:
 
-```PowerShell
-Install-Package Azure.Identity
+ - Recognizing form fields and content, using custom models trained to recognize your custom forms.  These values are returned in a collection of `RecognizedForm` objects.
+ - Recognizing form content, including tables, lines and words, without the need to train a model.  Form content is returned in a collection of `RecognizedPage` objects.
+ - Recognizing common fields from US receipts, using a pre-trained receipt model on the Form Recognizer service.  These fields and meta-data are returned in a collection of `RecognizeReceipt` objects.
+
+### FormTrainingClient
+
+`FormTrainingClient` provides operations for:
+
+- Training custom models to recognize all fields and values found in your custom forms.  A `CustomFormModel` is returned indicating the form types the model will recognize, and the fields it will extract for each form type.
+- Training custom models to recognize specific fields and values you specify by labeling your custom forms.  A `CustomFormModel` is returned indicating the fields the model will extract, as well as the estimated accuracy for each field.
+- Managing models created in your account.
+
+Please note that models can also be trained using a graphical user interface such as the [Form Recognizer Labeling Tool][labeling_tool].
+
+### Long-Running Operations
+
+Because analyzing and training form documents takes time, these operations are implemented as [**long-running operations**][dotnet_lro_guidelines].  Long-running operations consist of an initial request sent to the service to start an operation, followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has succeeded, to get the result.
+
+For long running operations in the Azure SDK, the client exposes a `Start<operation-name>` method that returns an `Operation<T>`.  You can use the extension method `WaitForCompletionAsync()` to wait for the operation to complete and obtain its result.  A sample code snippet is provided to illustrate using long-running operations [below](#extracting-receipt-values-with-a-long-running-operation).
+
+## Examples
+The following section provides several code snippets illustrating common patterns used in the Form Recognizer .NET API.
+
+* [Recognize Receipts](#recognize-receipts)
+* [Recognize Content](#recognize-content)
+* [Recognize Custom Forms](#recognize-custom-forms)
+* [Train a Model](#train-with-forms)
```

Comment:
should we add here that this is an example of our async methods too?

Reply:
hmm I'm not sure anymore. If something it would be in the manage custom forms section...
Not sure if there will be value to show that on those.
Maybe for now just leave as is

Reply:
I'll leave as-is for now, but worth coming back to before GA.  Tracking with issue: https://github.com/Azure/azure-sdk-for-net/issues/11475